src/constants/candles.ts

```diff
@@ -60,6 +60,20 @@ export enum CandleResolution {
   ONE_DAY = '1DAY',
 }
 
+/**
+ * @description ResolutionStrings used with TradingView's charting library mapped to time interval per candle in ms
+ */
+export const RESOLUTION_TO_INTERVAL_MS = {
+  '1': timeUnits.second,
+  '5': 5 * timeUnits.minute,
+  '15': 15 * timeUnits.minute,
+  '30': 30 * timeUnits.minute,
```

Comment:
nit, unrelated to this PR specifically, I wonder if it'd be better to convert `CandleResolution`, `RESOLUTION_MAP`, and `RESOLUTION_CHART_CONFIGS` to functions where we swtich case on the keys (that we could convert to an enum) just to make sure we catch all the cases. But honestly I don't imagine this code will change too much so just a light consider

Reply:
i like all of these as enums! nice way to get free typing so people don't put in something crazy (unless resolutionStrings already gives us that? that type has a few layers of abstraction to it so it's not immediately clear what it refers to

Reply:
ahh yeah, there's actually a way we can do that with just typescript by converting the library's `ResolutionString` (which is unfort just a generic string type) to an enum with the same values and changing the type of these maps. for example if you do:

```
enum InterfaceResolutionString {
  ONE_MINUTE = '1',
  FIVE_MINUTES = '5',
  FIFTEEN_MINUTES = '15',
  THIRTY_MINUTES = '30',
  ONE_HOUR = '60',
  FOUR_HOURS = '240',
  ONE_DAY = '1D',
}

/**
 * @description ResolutionStrings used with TradingView's charting library mapped to time interval per candle in ms
 */
export const RESOLUTION_TO_INTERVAL_MS: { [resolution in InterfaceResolutionString]: number } = {
  '1': timeUnits.second,
  '5': 5 * timeUnits.minute,
  '15': 15 * timeUnits.minute,
  '30': 30 * timeUnits.minute,
  '60': timeUnits.hour,
  '240': 4 * timeUnits.hour,
  // '1D': timeUnits.day,
};
```
you'll see that this gives a typescript error because the key `1D` is missing. we can address this separately if we'd like though :P 

we can alternatively also combine all three maps into one map, so something like

```
type ResolutionConfig = { candleResolution: CandleResolution, intervalMs: number, defaultRange: number }
export const RESOLUTION_MAP: { [resolution: ResolutionString]: ResolutionConfig } = {
  '1': { candleResolution: CandleResolution.ONE_MINUTE, intervalMs: timeUnits.second, defaultRange: timeUnits.hour },
  ...
};

```

Reply:
yeahhh that's very much in line with what i was envisioning. i dont think either have to be done in this PR but good to keep in mind for a cleanup task 👍